docs/content/preview/explore/query-1-performance/pg-stat-statements.md

```diff
@@ -304,6 +305,67 @@ ed) desc limit $1
 (7 rows)
 ```
 
+### View execution latencies for specific time spans
```

Comment:
1. Move this section before the Examples  
2. Change the title to "yb_latency_histogram column description"